jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import numpy.ma as ma
+
+from traitlets import List, Unicode, Bool, Int, observe
+
+from astropy.table import QTable
+from astropy.coordinates import SkyCoord
+from astroquery.sdss import SDSS
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+
+__all__ = ['Catalogs']
+
+
+@tray_registry('imviz-catalogs', label="Imviz Catalogs")
```

Comment:
I know most of the other plugins all say "Imviz", but in theory we might extend this to cubeviz/mosviz in the future, so a more generic name now might make that easier.  (We might also want to consider renaming `imviz-catalogs`, but that will then also need to be updated in `imviz.yaml`, so I won't include it in the suggestion here).

```suggestion
@tray_registry('imviz-catalogs', label="Catalog Search")
```

Reply:
I can bring up renaming the plugin in `imviz.yaml` on Monday!